marmelab/react-admin
# [Storybook] Re-enable storysource addon

## Problem

Storybook storysource addon (allowing to display the source code for a story in the 'Code' panel) was broken since we updated to Storybook v8.

It would be nice to re-enable it to allow viewers of our storybook to also view the matching source code.

## Solution

I actually managed to track down a [bug](https://github.com/storybookjs/storybook/pull/29669) in Storybook, preventing this addon to work in the first place.

This PR re-enables the plugin, and is to be merged when the issue is fixed in a new storybook release.

**Blocked:** wait until https://github.com/storybookjs/storybook/pull/29669 is merged and released
**=> EDIT:** No news from storybook maintainers, so this PR uses a local patch instead.

## How To Test

Run storybook, and check the source code of each story is present in the 'Code' panel.

https://react-admin-storybook-i85occt9g-marmelab.vercel.app/?path=/story/ra-core-form-filterliveform--basic
